lib/src/slices/api_knowledge/model/fingerprint/api_knowledge_model_fingerprint_req.dart

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) TIKI Inc.
+ * MIT license. See LICENSE file in root directory.
+ */
+
+class ApiKnowledgeModelFingerprintReq {
+  Set<String>? fingerprints;
+
+  ApiKnowledgeModelFingerprintReq({this.fingerprints});
+
+  ApiKnowledgeModelFingerprintReq.fromJson(Map<String, dynamic>? json) {
+    if (json != null) {
```

Comment:
Instead of checking null here, it could just ask for non-nullable input `ApiKnowledgeModelFingerprintReq.fromJson(Map<String, dynamic> json)`